swap/swap.go

```diff
@@ -198,6 +198,7 @@ const (
 	balancePrefix          = "balance_"
 	sentChequePrefix       = "sent_cheque_"
 	receivedChequePrefix   = "received_cheque_"
+	pendingChequePrefix    = "pending_cheque_"
```

Comment:
Can you add this new key to the test `TestStoreKeys`?

Reply:
done